src/core.js

```diff
@@ -516,16 +511,18 @@ function extractStacktrace( e, offset ) {
 }
 
 function sourceFromStacktrace( offset ) {
-	var e = new Error();
-	if ( !e.stack ) {
+	var error = new Error();
+
+	// Ref https://github.com/jquery/qunit/issues/636
```

Comment:
How about this?

```js
// Not all browsers generate the `stack` property for `new Error()`, see also #636
```

Though that makes me wonder which browsers those are - can we list them?